spec/system/candidate_interface/entering_details/candidate_viewing_a_science_gcse_spec.rb

```diff
@@ -1,89 +1,94 @@
 require 'rails_helper'
 
-RSpec.feature 'Candidate viewing Science GCSE', continuous_applications: false do
+RSpec.feature 'Candidate viewing Science GCSE', :continuous_applications, skip: 'Fix secondary courses' do
```

Comment:
Another skip until this bug is fixed: https://trello.com/c/QwRTwy2z/1140-bug-science-gcse-validation-on-secondary-courses